mantidproject/mantid
# Updated Muon GUI's to usage report and deprecation of old

**Description of work.**
This PR adds usage reporting to Muon Analysis, Elemental Analysis and Frequency Domain Analysis

It also renames the original Muon GUI and Muon Analysis 2.


**To test:**


Open MantidPlot
Open Muon Analysis (old) - it should be the original GUI
Open Muon Analysis - it should be the new GUI
Fixes #26774. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
